packages/engine/README.md

```diff
@@ -206,9 +206,81 @@ $ export RUST_LOG=debug
 [docs]: https://hash.ai/docs/simulation?utm_medium=organic&utm_source=github_readme_engine
 
 
+### Simulation Inputs
+> **WIP** - This section is a work-in-progress. More in-depth documentation is in the works for describing all input formats and options, and a project structure.
+
+#### Behavior keys
+Behavior keys define the **data type** of the fields that a behavior accesses on an agent's state. See the [docs](https://hash.ai/docs/simulation/creating-simulations/behaviors/behavior-keys?utm_medium=organic&utm_source=github_readme_engine) for an explanation for behavior keys in general. If you don't have a project set up in [hCore], it's also possible to create the behaviors keys by hand. Generally, every state variable requires a key. The top level dictionary consist of up to three fields: `"keys"`, `"built_in_key_use"`, and `"dynamic_access"`, while the latter two are neither required, nor used currently:
+
+```json
+{
+  "keys": {},
+  "built_in_key_use": null,
+  "dynamic_access": true
+}
+```
+
```

Comment:
```suggestion

> Note: JSON Objects have fields/members specified by [key]/[value] pairs. To avoid confusion between behavior _keys_ and the _keys_ in JSON Objects, we will be referring to the pairs as members or fields.

```